Owncloud iOs Client/DataBase/DTOs/UserDto.h

```diff
@@ -33,6 +34,8 @@ typedef enum {
 @property (nonatomic, copy) NSString *url;
 @property (nonatomic, copy) NSString *username;
 @property (nonatomic, copy) NSString *password;
```

Comment:
Since password will be contained in ```credDto```, ```password``` property needs to disappear.

Reply:
👍 Password already removed